Mathlib/Analysis/SpecialFunctions/NonIntegrable.lean

```diff
@@ -7,7 +7,7 @@ import Mathlib.Analysis.SpecialFunctions.Log.Deriv
 import Mathlib.MeasureTheory.Integral.IntervalIntegral.FundThmCalculus
 
 /-!
-# Non integrable functions
+# Nonintegrable functions
```

Comment:
dito regarding hyphens

Reply:
See general comment below.

Reply:
Either option seems well-established in literature. I went with "non-integrable".